gcc/config/arc/arc.h

```diff
@@ -1687,4 +1675,26 @@ enum
 			  && (arc_fpu_build == FPX_QK))
 #define TARGET_FP_ASSIST ((arc_fpu_build & FPX_DP) != 0)
 
+/* Handle pragmas for JLI calls. */
+#define REGISTER_TARGET_PRAGMAS() do {                                  \
+  c_register_pragma (0, "jli_call_fixed",  arc_pr_jli_call_fixed);      \
```

Comment:
The intent is for the code to be compatible with Metaware. It's not strictly necessary but is preferred.